src/socket-message-handling.ts

```diff
@@ -0,0 +1,127 @@
+import * as map from "lib0/map";
+
+const wsReadyStateConnecting = 0;
+const wsReadyStateOpen = 1;
+const wsReadyStateClosing = 2; // eslint-disable-line
+const wsReadyStateClosed = 3; // eslint-disable-line
+
+/**
+ * Map froms topic-name to set of subscribed clients.
+ * @type {Map<string, Set<any>>}
+ */
+const topics = new Map();
+
+const pingTimeout = 30000;
+
+/**
+ * @param {any} conn
+ * @param {object} message
+ */
+const send = (conn, message) => {
+  if (
+    conn.readyState !== wsReadyStateConnecting &&
+    conn.readyState !== wsReadyStateOpen
```

Comment:
nit:, maybe easier to read as
```js
![wsReadyStateConnecting, 
  wsReadyStateOpen].includes(conn.readyState)
```

but I suspect your version is more performant